sdk/program/src/bignum.rs

```diff
@@ -0,0 +1,784 @@
+//! @brief Solana Rust-based BigNumber for bpf-programs
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use std::fmt;
+
+#[repr(C)]
+#[derive(
+    BorshSerialize,
+    BorshDeserialize,
+    BorshSchema,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Hash,
+    AbiExample,
+)]
+pub struct BigNumber {
+    value: Vec<u8>,
+    negative: bool,
+}
+
+// pub struct BigNumber(u64);
```

Comment:
left over?

Reply:
Fixed in a156563ebf54e8934b207cb12cb2d8aefb83b93d